README_LINUX.md

```diff
@@ -213,8 +212,8 @@ This assumes your ccache executables are installed into `/usr/lib/ccache` - you
 Building a Debian package
 -------------------------
 
-The most up-to-date debian packaging rules are maintained by the
-Debian Multimedia team. Repository (with debian/ folder):
+The most up-to-date Debian packaging rules are maintained by the
+Debian Multimedia team. Repository (with Debian/ folder):
```

Comment:
this should stay lowercase debian/ since it's a directory name

Reply:
i will fix it immediately @snappizz thank you so much for your feedback